packages/fe/components/cid-table.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="cid-table">
+
+    <div class="toolbar">
+    </div>
+
+    <div class="table">
+      <CIDCard
+        v-for="(cid, i) in mockCids"
+        :key="`cid-${i}`"
+        :cid-data="cid"
+        :mobile="mobileTable" />
+    </div>
+
+    <div class="pagination">
+    </div>
+
+  </div>
+</template>
+
+<script>
+// ====================================================================== Import
+import CloneDeep from 'lodash/cloneDeep'
+
+import CIDCard from '@/components/cid-card'
+
+// =================================================================== Functions
+const handleTableResize = (instance) => {
+  if (window.matchMedia('(max-width: 64rem)').matches) {
+    if (!instance.mobileTable) {
+      instance.mobileTable = true
+    }
+  } else {
+    if (instance.mobileTable) {
+      instance.mobileTable = false
+    }
+  }
+}
+
+// ====================================================================== Export
+export default {
+  name: 'CIDTable',
+
+  components: {
+    CIDCard
+  },
+
+  data () {
+    return {
+      mobileTable: false,
+      resize: false,
+      cids: [
+        {
```

Comment:
use real api

Reply:
@joannaong we don't have an endpoint for cids just yet, but I moved this into a store action called after the dataset is fetched in the store